### PR TITLE
Support railway=border

### PR DESCRIPTION
**Pull request by Nakaner**
`railway=border` tag has been invented over a year ago but has not been implemented yet.

### CODE REVIEW

File: styles/standard.mapcss [hunk 1]
```diff
@@ -816,10 +816,11 @@ node|z14-[railway=tram_stop]
 /*****************/
 /* owner changes */
 /*****************/
-node|z11-[railway=owner_change]
+node|z11-[railway=border],
```
Comment (rurseekatze) [inline styles/standard.mapcss]: Why do you use one icon for both border and owner change? You created different icons, so they should be used.

File: josm-presets/de.xml [hunk 2]
```diff
@@ -1740,16 +1740,52 @@ See http://wiki.openstreetmap.org/wiki/OpenRailwayMap for details.
...
+				text="- railway:name:(abbrevation of operator 1) = (abbrevation of this location)"
+				de.text="- railway:ref:(Kürzel Betreiber 1) = (Abkürzung)" />
+			<label
+				text="- railway:name:(abbrevation of operator 2) = (abbrevation of this location)"
+				de.text="- railway:ref:(Kürzel Betreiber 2) = (Abkürzung)" />
+		</item>
+		<item name="Border" de.name="Grenzpunkt" icon="presets/douane.png" type="node">
+			<!-- presets/douane.png is included in JOSM's internal preset set -->
+			<label
+				text="Change of owner/operator near a boundary (admin_level>=2)"
```
Comment (rurseekatze) [inline josm-presets/de.xml]: Shouldn't it be `admin_level<=2` instead of `admin_level>=2`?

File: styles/standard.mapcss (hunk 1 above)
Comment (Nakaner) [inline styles/standard.mapcss]: I wrote at [the commit message](https://github.com/rurseekatze/OpenRailwayMap/pull/480/commits/dbf2246c02999f60b158641bb1da646cd00762b2):
> It is intention to use the same icon for both owner changes and borders.
The grey-black icon would be bad visible on the map.

File: josm-presets/de.xml [hunk 3]
```diff
@@ -1740,16 +1740,52 @@ See http://wiki.openstreetmap.org/wiki/OpenRailwayMap for details.
 				/>
 		</item>
 		<separator />
-		<item name="Grenzpunkt" icon="owner-change.png" type="node">
-			<label text="Eigentumswechsel der Infrastruktur an Privatbahnanschlüssen und Landesgrenzen" />
+		<item name="Change of Operator" de.name="Betreiberwechselpunkt" icon="owner-change.svg" type="node">
```
Comment (rurseekatze) [inline josm-presets/de.xml]: The term "Betreiberwechselpunkt" sounds a bit strange, I have never heard of this. I would propose the terms "Betreiberwechsel", "Betreibergrenze" or "Anschlussgrenze" instead.

File: josm-presets/de.xml [hunk 4]
```diff
@@ -1740,16 +1740,52 @@ See http://wiki.openstreetmap.org/wiki/OpenRailwayMap for details.
...
+			<!-- presets/douane.png is included in JOSM's internal preset set -->
+			<label
+				text="Change of owner/operator near a boundary (admin_level>=2)"
+				de.text="Eigentumswechsel der Infrastruktur nahe der Staatsgrenze" />
+			<key key="railway"
+				value="border" />
+			<space />
+			<label
+				text="You have to add manually following tags:"
+				de.text="Manuell sind noch folgende Tags zu ergänzen:" />
```
Comment (rurseekatze) [inline josm-presets/de.xml]: Could you use the syntax convention we already use, please? The first attribute should be in the same line with the label element.

File: styles/standard.mapcss (hunk 1 above)
Comment (rurseekatze) [inline styles/standard.mapcss]: Sorry, I did not read that. It is not the best solution to use the same icon for multiple features, but at the moment it is ok as a quick solution. We can think of an alternative icon in the future.

File: josm-presets/de.xml (hunk 2 above)
Comment (Nakaner) [inline josm-presets/de.xml]: fixed

**Comment by Nakaner** [comment]
I have fixed all the issues @rurseekatze complained about.

**Pull request merged**